### ISSUE TITLE
compiler throw related to rulesDST

### ISSUE THREAD

**Issue by buffetboy2001** [andydoro/DST_RTC#1]
Thanks for the handy library and the Word Matrix Clock project. Looks like I uncovered a bug in v1.1.X of your code; it throws when Arduino IDE v1.8.13 builds. I have not spent enough time with it to provide a resolution. The compiler fails due to an "undefined reference to `rulesDST`". Clearly it is declared in the header on line 5. So, not sure right now. 

https://github.com/andydoro/DST_RTC/blob/e95669478845c6b291fd0b6bb24a53dbd0937a20/DST_RTC.h#L5

This occurs when building on Windows and Mac. On both, if I roll back the library to v1.0.0, the code (Word Clock code) builds.

**Comment by andydoro** [comment]
Yes, due to an update you now need to explicitly declare whether using US or EU DST rules.

`// Define US or EU rules for DST comment out as required. More countries could be added with different rules in DST_RTC.cpp
const char rulesDST[] = "US"; // US DST rules
// const char rulesDST[] = "EU";   // EU DST rules`

It could be nice to have the library backwards compatible and use US as default if no rules are declared.

**Comment by andydoro** [comment]
New version has been created which is backwards compatible with v1.1.0.
